docs/_Docs_Pots.md

```diff
@@ -0,0 +1,58 @@
+
+Assigning funding to pots is purely a bookkeeping mechanism for a colony. From the perspective of the blockchain, ether and tokens are held by the colony contract until they are paid out when a task is completed.
+
+==TOC==
+
+## Types of Pots
+A colony will have many pots, but at a minimum will have one pot for rewards ( `pots[0]` ) and one for working capital ( `pots[1]` ), both of which are associated with the colony-wide domain.
+
+### Rewards
+Every colony has a special pot, `pots[0]`, which accrues funds by taking a small percentage of colony revenue. Members of the colony may claim rewards from this pot based on the number of colony tokens they have.
+
+When funds are sent to a colony as revenue, they must be put into the working capital pot before they can be further distributed to relevant sub-domains. In doing so, 10% of the revenue is siphoned off and put into the _rewards_ pot.
+
+When triggered, the rewards pot disburses to all members of the colony that have *both* tokens and reputation. Rewards are limited to the (normalized) geometric average of token holdings and reputation score (read more about the rewards formula in section 10.2 of the colony whitepaper).
+
+The rewards mechanism maximizes payout for individuals who both contribute meaningful work to the colony (evidenced by their reputation), and who maintain 'skin in the game' (evidenced by their unsold token holdings).
+
+### Domain Funding
+When new domains are created, they each are assigned a newly created pot, which can then be funded from the parent domain (see 'Funding Proposals' below). Pots associated with domains may only pay out to other pots.
+
+### Task Funding
+Each task created within a domain also has its own pot, which is funded from the domain to which the task belongs. Pots associated with tasks may only pay out to the individuals associated with the task (the Manager, Evaluator, and Worker of the task).
+
+## Funding Proposals
+Funding proposals are the mechanism that mediates the flow of funds between pots. Funding proposals are created by any user with sufficient reputation to stake in the relevant domain.
+
+A funding proposal consists of the following variables:
+* `Creator` - The individual that created the proposal
+* `From` - Pot funds are coming from
+* `To` - Pot funds are going to
+* `TokenType` - The token contract address
+* `CurrentState` - The state of the proposal (`inactive`, `active`, `completed`, or `cancelled`)
+* `TotalPaid` - How much has been transferred along this funding proposal so far.
+* `TotalRequested` - The maximum amount to transfer after which this funding proposal is considered `completed`.
+* `LastUpdated` - The time when the funding proposal was last updated
+* `Rate` - the rate of funding
+* `PunishCreator` - Whether the creator should be punished upon completion.
+
```

Comment:
too much details here, we should really have less technical spec in this level of documentation. Also Funding proposals are out of scope for the MVP so final structure may change